src/main/java/org/jenkinsci/plugins/configfiles/buildwrapper/ConfigFileBuildWrapper.java

```diff
@@ -102,42 +100,23 @@ public boolean isApplicable(AbstractProject<?, ?> item) {
 
     }
 
-    /**
-     * fix for JENKINS-17555
-     */
-    public class ManagedFilesEnvironment extends Environment {
```

Comment:
I agree with @jglick - I don't think its used by any other plugin, the only one I know which could be in question is the 'managed-scripts-plugin', but that one is not using it